### PR TITLE
lsn50v2 splitted topics more

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/dragino/lsn50v2-d20/uplink.js [hunk 1]
```diff
@@ -32,8 +32,11 @@ function consume(event) {
       data.extTrigger = !!(bytes[6] & 0x01);
       data.open = !!(bytes[6] & 0x80);
     }
+    emit("sample", { data, topic: "default" });
   }
 
+  data = {};
```
Comment (mariusgiger) [inline types/dragino/lsn50v2-d20/uplink.js:38]: do not reuse variables, instead locally scope the variable

File: types/dragino/lsn50v2-d22/uplink.js [hunk 2]
```diff
@@ -32,8 +32,11 @@ function consume(event) {
       data.extTrigger = !!(bytes[6] & 0x01);
       data.open = !!(bytes[6] & 0x80);
     }
+    emit("sample", { data, topic: "default" });
   }
 
+  data = {};
```
Comment (mariusgiger) [inline types/dragino/lsn50v2-d22/uplink.js:38]: do not reuse variables, instead locally scope the variable

File: types/dragino/lsn50v2-d23/uplink.js [hunk 3]
```diff
@@ -32,8 +32,11 @@ function consume(event) {
       data.extTrigger = !!(bytes[6] & 0x01);
       data.open = !!(bytes[6] & 0x80);
     }
+    emit("sample", { data, topic: "default" });
   }
 
+  data = {};
```
Comment (mariusgiger) [inline types/dragino/lsn50v2-d23/uplink.js:38]: do not reuse variables, instead locally scope the variable

File: types/dragino/lsn50v2-s31/uplink.js [hunk 4]
```diff
@@ -32,8 +32,11 @@ function consume(event) {
       data.extTrigger = !!(bytes[6] & 0x01);
       data.open = !!(bytes[6] & 0x80);
     }
+    emit("sample", { data, topic: "default" });
   }
 
+  data = {};
```
Comment (mariusgiger) [inline types/dragino/lsn50v2-s31/uplink.js:38]: do not reuse variables, instead locally scope the variable

File: types/dragino/lsn50v2-s31b/uplink.js [hunk 5]
```diff
@@ -32,8 +32,11 @@ function consume(event) {
       data.extTrigger = !!(bytes[6] & 0x01);
       data.open = !!(bytes[6] & 0x80);
     }
+    emit("sample", { data, topic: "default" });
   }
 
+  data = {};
```
Comment (mariusgiger) [inline types/dragino/lsn50v2-s31b/uplink.js:38]: do not reuse variables, instead locally scope the variable

**Review by mariusgiger — CHANGES_REQUESTED** [review]
looks good only a minor comment

**Review by sevyakenza — APPROVED** [review]

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**